debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/type/connect/ConnectBytesType.java

```diff
@@ -31,20 +31,20 @@ public String[] getRegistrationKeys() {
     }
 
     @Override
-    public String getDefaultValueBinding(DatabaseDialect dialect, Schema schema, Object value) {
-        return String.format(dialect.getByteArrayFormat(), ByteArrayUtils.getByteArrayAsHex(value));
+    public String getDefaultValueBinding(Schema schema, Object value) {
+        return String.format(getDialect().getByteArrayFormat(), ByteArrayUtils.getByteArrayAsHex(value));
     }
 
     @Override
-    public String getTypeName(DatabaseDialect dialect, Schema schema, boolean key) {
+    public String getTypeName(Schema schema) {
         final int columnSize = Integer.parseInt(getSourceColumnSize(schema).orElse("0"));
+        DatabaseDialect dialect = getDialect();
         if (columnSize > 0) {
-            return dialect.getTypeName(Types.VARBINARY, Size.length(columnSize));
+            return dialect.getJdbcTypeName(Types.VARBINARY, Size.length(columnSize));
         }
-        else if (key) {
-            return dialect.getTypeName(Types.VARBINARY, Size.length(dialect.getMaxVarbinaryLength()));
+        else {
+            return dialect.getJdbcTypeName(Types.VARBINARY, Size.length(dialect.getMaxVarbinaryLength()));
```

Comment:
This will present the same issue as the SQL Server dialect with `VARBINARY` as 8000 vs MAX.

Reply:
wdym?
It's the same code as before (with the new method name that collide with the same name used in here from another class)

Reply:
It is now after adding the `isKey` argument.